### DISCUSSION TITLE
27 Nov - 3 Dec 2023 BPD Weekly Update

### DISCUSSION THREAD

**Discussion by kjaymiller** [BlackPythonDevs/blackpythondevs#23] [Announcements]
I hope everyone had a great week last week. As we get ready to wrap up the year I hope that we can continue to progress forward on some of our goals and make sure that we're taking time to connect with family and friends as well.

## Jay Back in Office

I'm back and appreciate folks giving me plenty of time to relax and recharge with family. The next couple weeks are a little sporadic for me with some work items but hopefully I'll be able to get a little bit accomplished for BPD this week.

## Github Projects

I didn't do much work on this but I do have some updates that I can add to the [GitHub Project](https://github.com/orgs/BlackPythonDevs/projects/1)

## Black Python Devs Brand and Design

I heard back from two of the design teams. I would like to chat with them and @Veldakiara about their ideas and get a quote. Velda if you can DM me your availability in the next couple weeks I can reach out to them to schedule a call (or we can work via email).

## YouTube Content

@oleksis, I saw that you created some videos. Again I would love for us to add some video content and share about it on our social media. That being said I haven't stopped to watch the videos yet but trust if you think they are good to add to the channel let me know. 

Also I would love to add you and Velda both as moderators of the YouTube channel which would allow you to publish recordings for the community to the YouTube channel as it makes sense to. If you can DM me your youtube accounts so I can add them to the org.

## Back for Friday

This Friday Coffee and Code should be a relaxing one. We've been doing a lot and it would be good to take that time to slow down and look at some PRs and maybe answer some questions from the community.

## Relaxation is Wonderful

Make sure that you're taking time to relax and recharge. Things are going to slow down a lot over the holidays so take advantage of that time.

**Comment by VeldaKiara** [comment]
Hi @kjaymiller, welcome back.

I am available whenever you are.

Setting up the event in a bit

**Reply by kjaymiller** [reply]
I'll reply to them later today to schedule call

**Reply by VeldaKiara** [reply]
awesome

**Comment by oleksis** [comment]
Welcome back @kjaymiller . We miss you!
I'm glad you enjoy the time you deserve with the Family.
Sending a DM with my YouTube account
